_api-reference/index-apis/component-template.md

```diff
@@ -0,0 +1,152 @@
+---
+layout: default
+title: Create or update component template
```

Comment:
```suggestion
title: Create component template
```

Reply:
I think we need update in here so that its clear to people that the same endpoint performs both functions.